api/CredentialsContainer.json

```
@@ -189,7 +189,7 @@
               "version_added": null
```

I checked this out and it's also unsupported on desktop

```suggestion
              "version_added": false
```